src/plugins/intel_gpu/include/intel_gpu/graph/kernel_impl_params.hpp

```diff
@@ -58,6 +58,8 @@ struct kernel_impl_params {
     std::vector<std::shared_ptr<program>> inner_progs = {};
     std::vector<std::shared_ptr<network>> inner_nets = {};
     std::vector<std::map<size_t, primitive_id>> io_output_maps = {};
+    std::vector<size_t> output_size;
+    std::vector<size_t> img_size;
```

Comment:
Currently these values are only used for prior_box. It may cause confusion from in/output_layouts. So please add comments like 
// TODO : Thsese values are temporarily added for prior box. Such values decided at runtime shape infer and primitive creation will be handled with more generalized way in the near future.

Reply:
Why is it needed at all? Can't values of the corresponding tensors be read when it's needed instead of this WA?

Reply:
Umm yeah we can read again in the ocl impl creation for these values but it will affect performance. Isn't it better to read and store one time at calc_output_layout and then use them at ocl impl creation?

Reply:
There are such values read or calculated at calc_output_layout such as crop's offset and these values. In my thought, we can store such "runtime parameters" as a binary format in impl_param and then parse it at each primitive type on demand.

Reply:
Updated comment. Thanks!